test/dhcp_isc/server_isc_test.rb

```diff
@@ -40,7 +39,7 @@ def test_isc_provider_initialization
     ::Proxy::DhcpPlugin.load_test_settings(:server => 'a_server')
     ::Proxy::DHCP::ISC::Plugin.load_test_settings(:config => 'config_file', :leases => 'leases_file',
                                                   :omapi_port => '7777', :key_name => 'key_name',
-                                                  :key_secret => 'key_secret')
+                                                  :key_secret => 'key_secret', )
```

Comment:
This breaks the test on Ruby 1.8.7, but like the previous comment, I don't think this file needs any changes.  Could you revert it to what's in develop please?